packages/render-fidelity-tools/src/components/rendering-scenario.ts

```diff
@@ -31,7 +31,7 @@ export class RenderingScenario extends LitElement {
 
   @property({type: Array}) exclude: Array<string> = [];
 
-  @property({type: Object}) analysis: any|null;
+  @property({type: Object}) analysis: ScenarioRecord|null = null;
```

Comment:
Correct me if im wrong, i think creating a new interface for this object type is a better practice in ts.